internal/connect/command_offset_status_describe.go.go

```diff
@@ -0,0 +1,90 @@
+package connect
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/kafka"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type alterStatusOut struct {
+	Id        string `human:"ID" serialized:"id"`
+	Phase     string `human:"Phase" serialized:"phase"`
+	Message   string `human:"Message,omitempty" serialized:"message,omitempty"`
+	AppliedAt string `human:"Applied At,omitempty" serialized:"applied_at,omitempty"`
+}
+
+func (c *offsetStatusCommand) newStatusDescribeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:         "describe",
+		Short:       "Describe connector offset update status",
```

Comment:
Please take a look at the linter errors. All descriptions need to end with a period.
```suggestion
		Short:       "Describe connector offset update status.",
```

Reply:
done